java/core/src/test/org/apache/orc/impl/TestConvertTreeReaderFactory.java

```diff
@@ -32,29 +34,105 @@
 import org.apache.orc.OrcFile;
 import org.apache.orc.Reader;
 import org.apache.orc.RecordReader;
-import org.apache.orc.TestVectorOrcFile;
+import org.apache.orc.TestProlepticConversions;
 import org.apache.orc.TypeDescription;
+import org.apache.orc.Writer;
+import org.junit.Before;
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.TestName;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.sql.Timestamp;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.GregorianCalendar;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
```

Comment:
Hi, @pgaref . Although this is a test file, the `import` statement order looks a little strange to me.

Reply:
Hey @dongjoon-hyun -- you are right, seems that we dont follow a consistent import layout across the project. Switched to java -> others -> static (that seems to be followed in most of the src files)